projects/components/src/stepper/tab/stepper-tab.component.ts

```diff
@@ -1,15 +1,19 @@
-import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+import { ChangeDetectionStrategy, Component, ContentChild, Input } from '@angular/core';
 import { AbstractControl } from '@angular/forms';
 import { IconType } from '@hypertrace/assets-library';
 
-import { ContentHolder, CONTENT_HOLDER_TEMPLATE } from '../content/content-holder';
+import { ContentHolder, CONTENT_HOLDER_TEMPLATE } from '../../content/content-holder';
+import { StepperTabControlsComponent } from '../tab-controls/stepper-tab-controls.component';
 
 @Component({
   selector: `ht-stepper-tab`,
   template: CONTENT_HOLDER_TEMPLATE,
   changeDetection: ChangeDetectionStrategy.OnPush
 })
 export class StepperTabComponent extends ContentHolder {
+  @ContentChild(StepperTabControlsComponent)
+  public tabControls?: StepperTabControlsComponent;
+
```

Comment:
Do we also want to define positions of buttons? If that is the case we can also go with just taking in templates as input. Eg. `backTemplate`, `nextTemplate`

Reply:
We could leave it for the consumer to decide what they want to show.